libraries/botbuilder-dialogs/tests/numberPrompt.test.js

```diff
@@ -249,4 +250,92 @@ describe('NumberPrompt', function () {
             .send('0')
             .assertReply('ok')
     });
+
+    it('should consider culture specified in constructor', async function () {
+        // Initialize Testawait adapter.
```

Comment:
Testawait adapter?

Reply:
Hmmm...definitely copied the first unit test as a scaffoled for other tests.

Removed the offending comments!